src/cs2cs.c

```diff
@@ -293,10 +294,15 @@ int main(int argc, char **argv)
               case 's': /* reverse output */
                 reverseout = 1;
                 continue;
-              case 'd': /* set debug level */
+              case 'D': /* set debug level */
                 if (--argc <= 0) goto noargument;
                 pj_ctx_set_debug( pj_get_default_ctx(), atoi(*++argv));
                 continue;
+              case 'd':
+                if (--argc <= 0) goto noargument;
+                snprintf(oform_buffer, sizeof(oform_buffer), "%%.%df", atoi(*++argv));
```

Comment:
The use of `snprintf` is what is causing Travis to trip up. Unfortunately `snprintf` is not C89  which is a requirement at the moment. While a bit more cumbersome it should be possible to do this with `sprintf` instead.

Reply:
Oh ... that was the reason :) thanks.